raft-tech/TANF-app
# correction on backend URLs in README and add network policy for internal route

## Summary of Changes
With Nginx new configuration merged into develop, the frontend app need to use internal route to connect to backend and  we will need to add the correct network policy to let that port to open in the backend and accept communication. We had to add the following for qasp:

```
cf add-network-policy tdp-frontend-qasp tdp-backend-qasp --protocol tcp --port 8080
```

We will need to add similar config in the deployment script to automate the process for all apps.

Additionally, the [TDP-environment-README](https://github.com/raft-tech/TANF-app/blob/develop/docs/Technical-Documentation/TDP-environments-README.md) file needs revision to reflect correct backend URLS
## How to Test
Deploy to an environment - make sure the network policy is deleted before deployment using ``` cf remove-network-policy $APP``` and then list network-policies using:
```
cf network-policies
```

## Deliverables
_More details on how deliverables herein are assessed included [here](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverables)._

### [Deliverable 1: Accepted Features](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-1-Accepted-Features)

Checklist of ACs:

### [Deliverable 2: Tested Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-2-Tested-Code)

+ Are all areas of code introduced in this PR meaningfully tested?
+ Are code coverage minimums met?

### [Deliverable 3: Properly Styled Code](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-3-Properly-Styled-Code)


### [Deliverable 4: Accessible](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-4-Accessibility)



### [Deliverable 5: Deployed](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-5-Deployed)


### [Deliverable 6: Documented](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-6-Code-documentation)


### [Deliverable 7: Secure](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-7-Secure)


### [Deliverable 8: User Research](../docs/How-We-Work/our-priorities-values-expectations.md#Deliverable-8-User-Research)

Research product(s) clearly articulate(s):
